tracker/models.py

```diff
@@ -21,13 +22,26 @@ class CustomUserManager(BaseUserManager):
     - create_superuser: Creates and returns a new superuser with the given email and password.
     """
 
-    def create_user(self, email: str = None, password: str = None, role: str = None) -> "CustomUser":
+    def create_user(self, email: str, password: str, role: str = None) -> "CustomUser":
         """
         Creates a new user
-        :param email: str = None
-        :param password: str = None
+        :param email: str
+        :param password: str
+        :param role: str = None
         :return: CustomUser
         """
+        if not email:
```

Comment:
For `email` and `password` validations don't make any sense because the python side will crash

Reply:
hmmm, ok.

should I validate email if available?

Reply:
yep you could